lib/Github/Api/Apps.php

```diff
@@ -5,28 +5,14 @@
 use Github\Api\AcceptHeaderTrait;
 
 /**
- * @link   https://developer.github.com/v3/integrations/
+ * @link   https://developer.github.com/v3/apps/
  * @author Nils Adermann <naderman@naderman.de>
  */
-class Integrations extends AbstractApi
+class Apps extends AbstractApi
 {
     use AcceptHeaderTrait;
 
     /**
-     * Configure the accept header for Early Access to the integrations api
-     *
-     * @see https://developer.github.com/v3/integrations/
-     *
-     * @return self
-     */
-    public function configure()
```

Comment:
I think we should keep BC here and just remove the `$this->acceptHeaderValue = 'application/vnd.github.machine-man-preview+json';` but keep the method as users of the lib are possibly calling this method